.circleci/config.yml

```diff
@@ -1,22 +1,93 @@
 version: 2.1
 
-executors:
-  validate:
-    resource_class: small
-    docker:
-      - image: circleci/circleci-cli:latest
+orbs:
+  utils: coda/utils@<<pipeline.parameters.dev-orb-version>>
+  orb-tools: circleci/orb-tools@10.0
+  bats: circleci/bats@1.0
+  shellcheck: circleci/shellcheck@2.0
+
+# Pipeline Parameters
+## These parameters are used internally by orb-tools.
+parameters:
+  run-integration-tests:
+    description: An internal flag to prevent integration test from running before a development version has been created.
+    type: boolean
+    default: false
+  dev-orb-version:
+    description: >
+      The development version of the orb to test.
+      A "dev:alpha" version must exist for the initial pipeline run.
+    type: string
+    default: "dev:alpha"
+
+commands:
+
+
 jobs:
-  validate:
-    executor: validate
+  integration-test-notify:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - checkout
+      - utils/notify:
+          on_success: true
+
+  integration-test-slack:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - checkout
+      - utils/slack-notify-waiting-for-approval:
+          SLACK_BOT_TOKEN: ${PUSH_REMINDER_BOT_TOKEN}
+
+  integration-test-cancel-older-jobs:
+    docker:
+      - image: cimg/base:stable
     steps:
       - checkout
-      - run:
-          name: "Validate ORB config"
-          command: circleci orb validate config.yml
+      - utils/cancel-older-awaiting-approvals
 
 workflows:
-  version: 2
-  commit_validation:
+  test-pack:
+    unless: << pipeline.parameters.run-integration-tests >>
+    jobs:
+      - orb-tools/pack 
+      - shellcheck/check:
+          dir: ./src/scripts
```

Comment:
the exclude in shellchecks are for shellcheck codes to ignore; so SC2148 means ignore if `#!/bin/bash`